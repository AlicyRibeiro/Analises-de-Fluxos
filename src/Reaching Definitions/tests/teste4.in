3 6
x = 1
y = x + 2
z = y * 3
x = z - 4
y = x / 5
z = y + 6
1 2
2 3
3 0