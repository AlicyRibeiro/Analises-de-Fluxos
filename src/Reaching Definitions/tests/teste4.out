Bloco 0:
  IN : { }
  OUT: { (x,0,0) (y,0,1) }
Bloco 1:
  IN : { (x,0,0) (y,0,1) }
  OUT: { (x,1,1) (y,0,1) (z,1,0) }
Bloco 2:
  IN : { (x,0,0) (x,1,1) (y,0,1) (z,1,0) }
  OUT: { (x,0,0) (x,1,1) (y,2,0) (z,2,1) }
